packages/backend/src/shared/utils.ts

```diff
@@ -91,3 +95,22 @@ export async function poll<T>(args: PollArgs<T>): Promise<T> {
     // eslint-disable-next-line no-constant-condition
   } while (true)
 }
+
+export function verifyApiSignature(ctx: Context, config: IAppConfig): boolean {
```

Comment:
Should we perhaps include a signature version check in here as well?

Reply:
Added an env var for the backend to set its signature version for the api signature :+1: